src/main/webapp/app/config/input.constants.ts

```diff
@@ -1,2 +1,2 @@
 export const DATE_FORMAT = 'YYYY-MM-DD';
-export const DATE_TIME_FORMAT = 'YYYY-MM-DDTHH:mm';
+export const DATE_TIME_FORMAT = 'HH:mm:ss DD/MM/YYYY';
```

Comment:
Better to confirm with @sfdsilva if the timestamp format is the correct

Reply:
in the mocks, I placed time first but I think @DanielFran is right. Follow standard.